mlir/include/mlir/Dialect/SCF/Transforms/Passes.td

```diff
@@ -154,4 +154,18 @@ def SCFForToWhileLoop : Pass<"scf-for-to-while"> {
   }];
 }
 
+def TestSCFUpliftWhileToFor : Pass<"test-scf-uplift-while-to-for"> {
```

Comment:
Test passes should be in the test folder I believe.

Reply:
done